functional_testing/Openvcloud/RESTful/testcases/cloudbroker/test01_accounts.py

```diff
@@ -0,0 +1,264 @@
+        self.assertEqual(response.status_code,get_code)
+
+        location = self.api.get_location()
+        data,response = self.user_api.cloudapi.cloudspaces.create(accountId=self.response.json(), location=location, access=user_data["username"])
+        self.assertEqual(response.status_code,cloudspace_code)
+
+        new_account_name = self.utils.random_string()
+        response=self.user_api.cloudapi.accounts.update(accountId=self.response.json(),name=new_account_name)
+        self.assertEqual(response.status_code, update_code)
+
+
+      
+    def test004_delete_account(self):
+        """ OVC-000
+        *Test case for deleting account.*
+
+        **Test Scenario:**
+
+        #. Create account [C1] .
+        #. Delete account [C1], should succeed.
+        #. Try to get [C1], should has destroyed status.
+        #. Delete account [C1] again, should fail.
+        """
+        self.lg.info("Delete account [C1], should succeed.")
+        response = self.api.cloudbroker.accounts.delete(self.response.json())
+        self.assertEqual(response.status_code, 200)
+        self.CLEANUP['accounts'].remove(self.response.json())
+
+        self.lg.info("Try to get [C1], should has destroyed status.")
+        response= self.api.cloudapi.accounts.get(self.response.json())
+        self.assertEqual(response.json()["status"],"DESTROYED")
+
+        self.lg.info("Delete account [C1] again, should fail.")
+        response = self.api.cloudbroker.accounts.delete(self.response.json())
+        self.assertEqual(response.status_code, 404)
+
+    def test005_delete_accounts(self):
+        """ OVC-000
+        *Test case for deleting multiple accounts.*
+
+        **Test Scenario:**
+
+        #. Create accounts [C1],[C2] , [C3] and [C4], should succeed .
+        #. Delete account [C1], should succeed.
+        #. Delete accounts [C1], [C2], should fail.
+        #. Delete [C2] and [C3] accounts using delete acoounts api, should succeed. 
+        #. Check that the four acounts deleted ,should succeed.
+        """
+        accounts=[self.response.json()]
+        for _ in range(3):
+            data,response=self.api.cloudbroker.accounts.create(self.user)
+            accounts.append(response.json())
+        response= self.api.cloudbroker.accounts.delete(accounts[0])
+        self.assertEqual(response.status_code, 200)
+        self.CLEANUP['accounts'].remove(self.response.json())
+        
+        response= self.api.cloudbroker.accounts.deleteAccounts([accounts[1],accounts[0]])
+        self.assertEqual(response.status_code, 404)
+
+        response= self.api.cloudbroker.accounts.deleteAccounts([accounts[2],accounts[3]])
+        self.assertEqual(response.status_code, 200)
+
+        for accountID in accounts:
+            response = self.api.cloudapi.accounts.get(accountID)
+            self.assertEqual(response.json()["status"],"DESTROYED")
+       
+    def test006_account_disable_and_enable(self):
```

Comment:
split different test cases in separate methods

Reply:
done